components/DeleteAccount/index.js

```diff
@@ -1,13 +1,20 @@
+import Link from "next/link";
+
 import Heading2 from "../Heading2";
+
 import styles from "./styles.module.css";
 
 const DeleteAccount = () => {
   return (
     <div className={styles.container}>
       <Heading2>Are you sure you want to delete your account?</Heading2>
       <div className={styles.buttonContainer}>
-        <button>Cancel</button>
-        <button>Delete</button>
+        <Link href="/settings">
+          <button>Cancel</button>
```

Comment:
Nice work! By default we'll want to use the `<a>` tag nested within the Next.js <Link> tag.

`<a>` are the fundamental way we navigate on the web.

Reply:
Changed! Thanks